nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -13,7 +13,7 @@ import "contracts/proxy/Upgradeable.sol";
 
 
 /**
-* @notice Contract holds policy data and locks fees
+* @notice Contract holds policy data and locks accrued policy fees until collection
 * @dev |v5.1.1|
```

Comment:
because some signatures changed - need to change major version, `v6.1.1`

Reply:
✔️